### ISSUE TITLE
distributions: add/mult and test coverage

### ISSUE THREAD

**Issue by davidovitch** [jonathf/chaospy#25]
Is it reasonable to expect ```dist.backend.Dist + 1``` works, but ```1 + dist.backend.Dist``` fails? I don't have any experience with overloading operators, but I would assume both left and right additions should be possible? Or am I missing something else here?
```python
>>> import chaospy as cp
>>> cp.Uniform() + 1
<chaospy.dist.operators.Add at 0x7fb40405e400> 
# but this fails
>>> 1 + cp.Uniform()
Traceback (most recent call last):
  File "chaospy/src/chaospy/dist/backend.py", line 372, in __radd__
    return add(X, self)
  File "chaospy/src/chaospy/dist/operators.py", line 134, in add
    return Add(A=A, B=B)
  File "chaospy/src/chaospy/dist/operators.py", line 34, in __init__
    _length=length, _advance=True)
  File "chaospy/src/chaospy/dist/backend.py", line 115, in __init__
    self.dependencies = self.G.run(self.length, "dep")[0]
  File "chaospy/src/chaospy/dist/graph.py", line 266, in run
    out = self(self.root)
  File "chaospy/src/chaospy/dist/graph.py", line 158, in __call__
    return self._call(*args, **kwargs)
  File "chaospy/src/chaospy/dist/graph.py", line 526, in dep_call
    out = dist._dep(self)
  File "chaospy/src/chaospy/dist/backend.py", line 345, in _dep
    if len(self)==1:
TypeError: only integer arrays with one element can be converted to an index
```
If that would help, I can go and create a bunch of tests for a few things like adding/multiplying distributions, and sent it to you as a PR. Do you think creating a ```tests/test_dist.py``` file would be the right way to go?

**Comment by jonathf** [comment]
No, that is a bug.

The problem was easy to find and fix. Should be solved in #26.

I have not created tests for it, it would be much appreciated. Make a PR against the branch `development`, and I will take a look at it.

**Comment by jonathf** [comment]
I consider this closed. If you do not consider this to be the case, feel free to re-open.
